src/test/java/org/eclipse/uprotocol/rpc/RpcTest.java

```diff
@@ -25,14 +25,20 @@
 package org.eclipse.uprotocol.rpc;
 
 import com.google.protobuf.Any;
+import com.google.protobuf.ByteString;
 import com.google.protobuf.Int32Value;
 import com.google.protobuf.InvalidProtocolBufferException;
 import com.google.rpc.Code;
 import com.google.rpc.Status;
 import org.eclipse.uprotocol.transport.builder.UAttributesBuilder;
-import org.eclipse.uprotocol.transport.datamodel.UPayload;
 import org.eclipse.uprotocol.uri.serializer.LongUriSerializer;
-import org.eclipse.uprotocol.v1.*;
+import org.eclipse.uprotocol.v1.UUri;
```

Comment:
why do we have the version in the package name. makes upgrading even messier

Reply:
This is to ensure that if/when we go to a non-backwards compatible version of uProtocol (ex. v1) that we have differentiate the data model between them like we do for uSubscription, uDiscovery, etc...